base/src/main/java/com/smartdevicelink/proxy/rpc/UnsubscribeVehicleData.java

```diff
@@ -111,10 +111,13 @@
  * 		<tr>
  * 			<td>externalTemperature</td>
  * 			<td>Boolean</td>
- * 			<td>The external temperature in degrees celsius</td>
+ * 			<td>The external temperature in degrees celsius. This parameter is deprecated starting RPCSpec 7.1, please see climateData.</td>

```

Comment:
@kostyaBoss On line 114, I got 7.1.0 instead of 7.1 using the RPC generator tool. I would replace the 7.1 with 7.1.0.